test/verify/check-sosreport

```diff
@@ -39,6 +39,7 @@ class TestSOS(MachineCase):
         b.wait_visible("#sos")
 
         with b.wait_timeout(360):
+            b.wait_attr_not_contains(".progress-bar", "style", "width: 0%")
```

Comment:
Testing for something to not be there is always fragile.  It would be more robust to get the real width of the element and assert that it is not zero.